cmake/schemes/url_sha1_boost.cmake.in

```diff
@@ -88,8 +89,9 @@ else()
 endif()
 
 if("@MSVC@" OR "@MINGW@")
```

Comment:
I have not experienced anything yet on MSVC but I think it will be safe to convert everything to 8.3Filename on WIN32.